### PR TITLE
Now al-khaser is able to pass 2 more checks: interrupt 0x3 and 0x2d

### PR DESCRIPTION
**Pull request by 0ssigeno**

### CODE REVIEW

File: qiling/os/windows/utils.py [hunk 1]
```diff
@@ -46,3 +46,17 @@ def is_file_library(string):
 
 def string_to_hex(string):
     return ":".join("{:02x}".format(ord(c)) for c in string)
+
+
+def find_size_function(ql, func_addr):
+    # We have to retrieve the return address position
+    code = ql.mem.read(func_addr, 0x100)
+    if b"\xc3" in code:
```
Comment (xwings) [inline qiling/os/windows/utils.py:54]: loop or something ?since its all the same

Comment (0ssigeno) [reply qiling/os/windows/utils.py]: 3dd389f should resolve the issue

File: qiling/os/windows/utils.py [hunk 2]
```diff
@@ -46,3 +46,11 @@ def is_file_library(string):
 
 def string_to_hex(string):
     return ":".join("{:02x}".format(ord(c)) for c in string)
+
+
+def find_size_function(ql, func_addr):
+    # We have to retrieve the return address position
+    code = ql.mem.read(func_addr, 0x100)
+    opcodes = [b"\xc3", b"\xc2", b"\xcb", b"\xca"]
```
Comment (xwings) [inline qiling/os/windows/utils.py:54]: opcode sounds funny. these are  interrupt handler ? or any proper name ?

Comment (0ssigeno) [reply qiling/os/windows/utils.py:54]: https://c9x.me/x86/html/file_module_x86_id_280.html
I think the name is correct

Comment (xwings) [reply qiling/os/windows/utils.py:54]: opcode is a type. but these opcode design for a purpose.

Comment (0ssigeno) [reply qiling/os/windows/utils.py:54]: I mean, what i'm trying to do with this function is to find the return opcode.
Maybe return_opcodes is a better name? Or as you wish, i don't really mind

Comment (xwings) [reply qiling/os/windows/utils.py:54]: Return from Procedure. we need a shorter name

Comment (xwings) [reply qiling/os/windows/utils.py:54]: from wikipedia
```
In computing, an opcode (abbreviated from operation code, also known as instruction machine code, instruction code, instruction syllable, instruction parcel or opstring) is the portion of a machine language instruction that specifies the operation to be performed
```

File: qiling/os/windows/dlls/kernel32/errhandlingapi.py [hunk 3]
```diff
@@ -110,14 +101,45 @@ def hook_RaiseException(ql, address, params):
...
+        ql.mem.write(double_pointer, pointer.to_bytes(4, "little"))
+        # arg
+        ql.stack_push(double_pointer)
+        # ret
+        ql.stack_push(ret)
+        # func
+        ql.stack_push(start)
+
+    def exec_standard_into(ql, into, user_data):
+        # fixme probably this works only with al-khaser.
```
Comment (xwings) [inline qiling/os/windows/dlls/kernel32/errhandlingapi.py:125]: "FIXME:" not fixme

**Comment by xwings** [comment]
So,

He did not comment. I will silently commit

**Pull request merged**